guide/concepts/chats/search.md

```diff
@@ -43,3 +43,6 @@ chat.search({
     end: '123123123133'
 });
 ```
+
+Search will stop after looking through ```1,0000``` events and then emit the ```$.search.pause``` event.
```

Comment:
inline code blocks can also be placed with just single back ticks. no need for triple.

Reply:
more = better